reporter/src/main/kotlin/reporters/CycloneDxReporter.kt

```diff
@@ -54,6 +57,24 @@ class CycloneDxReporter : Reporter {
     private fun mapHash(hash: org.ossreviewtoolkit.model.Hash): Hash? =
         enumValues<Hash.Algorithm>().find { it.spec == hash.algorithm.toString() }?.let { Hash(it, hash.value) }
 
+    private fun mapLicenseNamesToObjects(licenseNames: Collection<String>, origin: String, input: ReporterInput): List<License> {
```

Comment:
This line is too long, but this can be easily avoided by turning the function body into an expression like:

    private fun mapLicenseNamesToObjects(licenseNames: Collection<String>, origin: String, input: ReporterInput) =
        licenseNames.map { licenseName ->
            val spdxId = SpdxLicense.forId(licenseName)?.id

            // Prefer to set the id in case of an SPDX "core" license and only use the name as a fallback, also
            // see https://github.com/CycloneDX/cyclonedx-core-java/issues/8.
            License().apply {
                id = spdxId
                name = licenseName.takeIf { spdxId == null }
                licenseText = LicenseText().apply {
                    contentType = "plain/text"
                    text = input.licenseTextProvider.getLicenseText(licenseName)
                }
                extensibleTypes = listOf(ExtensibleType("ort", "origin", origin))
            }
        }